packages/cli/src/commands/createReporter.ts

```diff
@@ -45,39 +45,38 @@ export async function createReporter(
     case "verboseFileLog":
     case "vfl":
       return new VerboseFileLogReporter(logFile);
+  }
 
-    default:
```

Comment:
This should be viewed with ignore whitespace--it's just moving this outside the switch and changing the indentation